frontend/src/views/instructors/editable-instructors-list.js

```diff
@@ -1,9 +1,34 @@
 import React from "react";
 import { connect } from "react-redux";
-import { instructorsSelector, upsertInstructor } from "../../api/actions";
+import { Button } from "react-bootstrap";
+import {
+    instructorsSelector,
+    upsertInstructor,
+    deleteInstructor,
+} from "../../api/actions";
 import { InstructorsList } from "../../components/instructors";
 import { EditableField } from "../../components/edit-field-widgets";
 
+/**
+ * A cell that renders a button for deleting the instructor from that row.
+ *
+ * @param {*} props
+ * @returns a button which when clicked deletes the instructor.
+ */
+function DeleteButtonCell(props) {
+    const {
+        deleteInstructor,
+        original: { id: applicantId },
```

Comment:
De-structuring more than one level is hard to read (and generates confusing error messages). Refactor this to only destructure one level deep.